cmake/modules/hunter_config.cmake

```diff
@@ -44,9 +44,36 @@ macro(hunter_config)
 
   string(COMPARE NOTEQUAL "${_hunter_GIT_SUBMODULE}" "" _hunter_submodule_create)
   if(_hunter_submodule_create)
+    # get HUNTER_SUBMODULE_SOURCE_SUBDIR from CMAKE_ARGS
```

Comment:
I think it make sense to create function that will accept `CMAKE_ARGS` as input and return `_hunter_SUBMODULE_SOURCE_SUBDIR`.